data/sdo-howto-examples.txt

```diff
@@ -0,0 +1,188 @@
+TYPES: #howto-1 HowTo, estimatedCost, totalTime, tool, supply, steps, HowToSection, HowToStep, HowToDirection, HowToTip, afterMedia, beforeMedia, duringMedia
+
+PRE-MARKUP:
+
+<div>
+  <strong><span>Change a Flat Tire</span></strong>
+  <div>About <span>$20</span></div>
+  <div>About <span>30 minutes</span></div>
+  <div>Necessary Items:</div>
+  <div>Spare tire</div>
+  <div>Lug wrench</div>
+  <div>Jack</div>
+  <div>Wheel wedges</div>
+  <div>Flares</div>
+  <div id="steps">
+    <div>Preparation</div>
+    <div>
+      <div>
+        <span><img href="lit-flares.jpg" /></span>
+        Turn on your hazard lights and set the flares.
+      </div>
+      <div>
+        You're going to need space and want to be visible.</div>
+      <div>
+        <div>
+          Position your wheel wedges in front of the front tires if a rear tire is flat, or behind the rear tires if a front tire is flat.
+        </div>
+        <div>
+          You don't want the car to move while you're working on it.
+        </div>
+      </div>
+    </div>
+  </div>
+  <div id="steps">
+    <div>Raise the car</div>
+    <div>
+      <div>
+        Position the jack underneath the car, next to the flat tire.
+      </div>
+    </div>
+    <div>
+      <div>
+        <span><img href="car-on-ground.jpg" /></span>
+        Raise the jack until the flat tire is just barely off of the ground.
+        <span><img href="car-raised.jpg" /></span>
+      </div>
+      <div>
+        It doesn't need to be too high.
+      </div>
+    </div>
+    <div>
+      <div>
+        Remove the hubcap and loosen the lug nuts.
```

Comment:
Lots of people would say that you should do this *before* raising the tire off the ground. It's much harder if the wheel is spinning…

I would have moved this step earlier.